src/probnum/filtsmooth/gaussfiltsmooth/kalman.py

```diff
@@ -99,20 +99,40 @@ def filter_step(self, start, stop, randvar, data, **kwargs):
         """
         data = np.asarray(data)
         predrv, _ = self.predict(start, stop, randvar)
-        filtrv, _, _, _ = self.update(stop, predrv, data)
+        filtrv, _, _ = self.update(stop, predrv, data)
         return filtrv
 
     def predict(self, start, stop, randvar, **kwargs):
         return self.dynamod.transition_rv(randvar, start, stop=stop, **kwargs)
 
     def update(self, time, randvar, data, **kwargs):
+        """
+
+        Parameters
+        ----------
+        time
+        randvar
+        data
+        kwargs
+
+        Returns
+        -------
+        Normal
```

Comment:
What do you think about adding the variable names here for additional clarity?
So:
`filt_rv : Normal` 
`meas_rv : Normal`
`info : Dict`

Reply:
fair enough. Tbh, I though this would somehow mess up sphinx, so I didn't do it. Not sure whether this is true, so I will add it now.

Reply:
Oh true I didn't check that, I just looked at the numpy styleguide and there it seems like an ok thing to do. Sorry if the suggestion might not work correctly :/